# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fwupd/hotfix-udev.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fwupd-1.8.0/meson.build.vanilla	2022-05-02 14:00:10.500329223 +0200
+++ fwupd-1.8.0/meson.build	2022-05-02 14:00:25.529328624 +0200
@@ -204,7 +204,7 @@
 endif
 gmodule = dependency('gmodule-2.0')
 if build_standalone
-gudev = dependency('gudev-1.0', version : '>= 232',
+gudev = dependency('gudev-1.0', version : '>= 132',
                     required: get_option('gudev').disable_auto_if(host_machine.system() != 'linux'))
 if gudev.found()
   conf.set('HAVE_GUDEV', '1')
